metrics/web/lib/base/presentation/widgets/rive_animation.dart

```diff
@@ -75,6 +75,15 @@ class _RiveAnimationState extends State<RiveAnimation> {
     });
   }
 
+  @override
+  void didUpdateWidget(covariant RiveAnimation oldWidget) {
```

Comment:
There is no need for the `covariant` keyword since it is already defined in the superclass.
```suggestion
  void didUpdateWidget(RiveAnimation oldWidget) {
```